packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -41,6 +48,44 @@ interface EnqueueTaskParams {
   name: string;
 }
 
+/**
+ * A pino-based singleton logger for use within Google Cloud Functions.
+ * Provides a metric() method to log metrics-related data
+ */
+export class GCFLogger {
+  private static logger: pino.Logger;
+
+  /**
+   * Get the logger instance
+   * @returns a pino-based logger
+   */
+  public static get(): pino.Logger {
+    if (!this.logger) {
+      this.logger = this.initLogger();
+    }
+    return this.logger;
+  }
+
+  private static initLogger(
+    options?: pino.LoggerOptions,
+    dest?: pino.DestinationStream
```

Comment:
Are these options used anywhere?

Reply:
these params are used by the tests to inject a mock/custom destination. This could also be made public in the future if we wanted to support bots adding multiple destinations for their logs

Reply:
oh I see what you mean with the options param - right, I was using them in the tests before but not anymore. I can take that out